web-ng/src/app/components/map/map.component.ts

```diff
@@ -178,6 +178,21 @@ export class MapComponent implements AfterViewInit, OnDestroy {
     });
   }
 
+  private centerViewportOnMarker(options: google.maps.MarkerOptions): void {
+    if (options.position) {
+      this.map.panTo(options.position);
+    }
+  }
+
+  private setMapZoomDefault(): void {
+    if (
+      this.initialMapOptions.zoom &&
```

Comment:
Iiuc this will zoom out the map to the original zoom level, whereas we would like to zoom in closer to the point. Can we add a separate constant or param for the preferred "zoomed in" level? Default to 13?

Reply:
Sorry for the late reply. This would only zoom in if the current zoom level is less than 3, it will never zoom out because if current zoom is greater than 3, then the condition will be false. If we need zoom to be 13 on click, then this if statement would not be useful as we can just set the zoom value. Please let me know if that makes sense. Thanks!

Reply:
Hi @DakshSaini ! Zoom level 3 is at the scale of an entire country. It's the default zoom level shown when the map is opened. When a point is clicked, we'd like the map to zoom in to approx. the scale of a house or building, which is much greater (approx. 13). That's why I was suggesting using a different constant to specify the "zoomed in" zoom level vs. the default "zoomed out" level (`this.initialMapOptions.zoom`). Does that make sense?

Reply:
@gino-m, thank you for the feedback and explanation, and I totally agree! I have now made the necessary changes.